modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase

```diff
@@ -1546,6 +1546,162 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11650. User can checkout a product in a channel that has also stock in another channel, and the product's stock in another channel cannot be affected by this channel"
+	@priority = 5
+	test CanCheckoutAProductInAChannelWithoutAffectingAnotherChannelStock {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And a second Minium site is created and buyer user is associated with it") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(
+				siteName = "Minium 2",
+				siteTemplateName = "Minium");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium 2",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And an Italy warehouse eligible for Minium Portal and a Southwest warehouse eligible for Minium 2 Portal are set") {
```

Comment:
I guess you are setting eligibility for Northeast for Minium 2 but wrote Southwest here:

And the Italy warehouse is set eligible for Minium Portal while the Northeast warehouse is set eligible for Minium 2 Portal